package/lib/notification/notification_entity.dart

```diff
@@ -20,17 +20,21 @@ class DetailsEntity {
 
 class NotificationEntity {
   final String notification;
-  final String identifier;
-  final String? reference;
   final String? notificationLogId;
+  final String? contactId;
+  final String? reference;
+  final String? userId;
+  final String? name;
```

Comment:
essa é a struct que vai renderizar o item na tela, realmente precisamos destes itens? 
Se for para a captura no click, não seria melhor ter um payload sendo um `Map<string, dynamic>`?